orchestration/values.yaml

```diff
@@ -27,3 +27,6 @@ argoTemplates:
     name: gsutil-upload
```

Comment:
You'll need to add an entry to this `argoTemplates` block to enable generation of the diff-tables `WorkflowTemplate`. See the latest clinvar-ingest.